src/backend/tests/Fusion.Resources.Api.Tests/Fixture/ResourceApiFixture.cs

```diff
@@ -74,18 +74,60 @@ internal ApiPersonProfileV3 AddProfile(Action<Testing.Mocks.ProfileService.Fusio
         internal ApiPersonProfileV3 AddResourceOwner(string department)
         {
             var resourceOwner = this.AddProfile(FusionAccountType.Employee);
-            resourceOwner.IsResourceOwner = true;
-            resourceOwner.FullDepartment = department;
-            resourceOwner.Department = new DepartmentPath(department).GetShortName();
-            LineOrgServiceMock.AddTestUser()
-                .MergeWithProfile(resourceOwner)
-                .WithFullDepartment(department)
-                .AsResourceOwner()
-                .SaveProfile();
 
+            SetAsResourceOwner(resourceOwner, department);
+            
             return resourceOwner;
         }
 
+        /// <summary>
+        /// Sets the profile as manager for the org unit. 
+        /// Returns the org unit created for the full department string.
+        /// </summary>
+        /// <param name="person"></param>
+        /// <param name="department"></param>
+        /// <returns></returns>
+        internal ApiOrgUnit SetAsResourceOwner(ApiPersonProfileV3 person, string department)
```

Comment:
Updated the fixture to correctly set up a manager. As this previous was calculated using `isResourceOwner` and `fullDepartment` it was easier to mock. Now we have to use fusion roles, with the SAP id, which is resolved through lineorg.